app/domain/person/address.rb

```diff
@@ -39,10 +39,12 @@ def combine_household_names(members)
 
   def address
     [
+      @person.address_care_of.to_s.strip.presence,
       @person.address.to_s.strip,
+      @person.postbox.to_s.strip.presence,
       [@person.zip_code, @person.town].compact.join(' ').squish,
       country
```

Comment:
Is `country` always present?

Reply:
tl;dr yes

The method `country` (a few lines below this one in the same class) abstracts this nicely, I think. If it's the default-country (as set in `config/settings.yml`), it is nil and this line is compacted away, otherwise, the country is rendered.

If it is indeed an empty string, it will also be rendered needlessly in the PDF. Luckily, whitespace does not waste too much ink/toner. :-)